app/views/good_job/processes/index.html.erb

```diff
@@ -27,7 +27,7 @@
           <div class="col">
             <code class="font-monospace">
```

Comment:
What would you think of also putting the full proctitle in a `title` attribute; similar to how relative timestamps also have the full timestamp available.

Reply:
👍 good call. Added in 2914ecc